frontend/libs/studio-components/src/components/StudioCenter/StudioCenter.module.css

```diff
@@ -2,6 +2,6 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  height: 100%;
-  width: 100%;
+  height: 100vh;
+  width: 100vw;
```

Comment:
`vh` og `vw` tilsvarer henholdsvis høyden og bredden på vinduet. Dette elementet vil altså ta opp like stor plass som hele vinduet, og det blir jo litt stort i de fleste tilfeller.
Hva er problemet som skal løses her? Hvis det er at innholdet ikke blir sentrert i sin boks, er dette noe som må løses der komponenten brukes ved å gi den nok plass.

Reply:
Er for å sentrere spinneren. Er hvertfall 3 tilfeller den er på toppen av skjermen:
- når man går fra dashboard til app-dev
- når lage-siden laster
- når tekst-siden laster

Men helt sikkert en bedre må å løse det på ja 😊

Reply:
Da må vi nok justere foreldreelementet i hvet av disse tilfellene. Problemet med `100vh` og `100vw` er at elementet _alltid_ vil ta opp like stor plass som vinduet, også når det egentlig bare skal vises på en liten del av skjermbildet.

Reply:
Har pushet en endring som forhåpentligvis svarer på det du foreslår ☺️